templates/src/graphql/schema.js

```diff
@@ -0,0 +1,51 @@
+const { gql } = require('apollo-server-express');
+
+const typeDefs = gql`
+
+    type User {
+      id: ID!
+      email: String!
+    }
+  
+    type PresignedUrl {
+      url: String
+      method: String 
+    }
+
+    type PresignedUrls {
+      upload: PresignedUrl
+      download: PresignedUrl
+    }
+
+    type Status {

```

Comment:
Can you please remove this from the schema? We shouldn't be exposing health check endpoints through GraphQL

Reply:
Removed status query from schema and resolver